LICENSE

```diff
@@ -0,0 +1,1170 @@
+The Open Computing Facility's directors and members believe strongly in
+the importance of free and open-source software, and we hope for
+software and other works developed by our organization to be used to
+advance the free software movement.
+
+We have spent a considerable amount of time discussing which licenses
+best reflect our organization's mission, with a roughly even split among
+members who believe that a copyleft license like the GPL best promotes
```

Comment:
We've included the entire text of the GPL v3, I don't think there's any ambiguity here...